### ISSUE TITLE
Unwanted characters around blue text with `nb --help'

### ISSUE THREAD

**Issue by naltun** [xwmx/nb#25]
**^O** is floating around all blue text when I run **nb --help**:
```shell
$ nb --help
^O__          _^O
^O\ \   _ __ | |__^O
^O \ \ | '_ \| '_ \^O
^O / / | | | | |_) |^O
^O/_/  |_| |_|_.__/^O
------------------^O

^O[nb]^O Command line note-taking, bookmarking, archiving with plain-text data
storage, encryption, filtering and search, Git-backed versioning and syncing,
Pandoc-backed conversion, global and local notebooks, customizable color
themes, plugins, and more in a single portable, user-friendly script.

^OHelp:^O
  nb help               Display this help information.
  nb help <subcommand>  View help information for <subcommand>.
  nb help --colors      View information about color settings.
  nb help --readme      View the `nb` README file.

^OUsage:^O
```

The **^O** cuddles only the blue text. I am unsure why this is.

Here is a screenshot:
<img width="553" alt="image" src="https://user-images.githubusercontent.com/7507990/95591906-eb1bf100-0a15-11eb-9312-dc19f87a6a99.png">

I am using **Alacritty 0.5.0** as my terminal emulator.

**Comment by xwmx** [comment]
Thanks for the heads up. It looks like this might be the issue: https://unix.stackexchange.com/a/446541

I'm currently unable to reproduce the issue with Alacritty 0.5.0 (a6681e3) on macOS. I'm still looking into this.

**Comment by xwmx** [comment]
For anyone encountering this, does it happen only in `tmux`, or does it also happen outside of `tmux`?

**Comment by xwmx** [comment]
This indeed appears to be due to the `less` quirk mentioned in that post. As far as I am aware, it only occurs in `tmux`. If anyone encounters it in any other contexts, please let me know. To resolve this, `nb` now checks to see if `$TMUX` is set and, if so, uses the modified `\033[m` control sequence rather than `tput sgr0`. As far as I can tell it doesn't cause any issues in `tmux`, but please let me know if it does.

This change is available in version 5.6.2.

Thanks!

**Comment by naltun** [comment]
Late to the comments -- thanks for your quick patch! Great project, congrats.
